plugins/notifications/back.js

```diff
@@ -39,13 +43,17 @@ module.exports = (win, options) => {
 				}
 				return;
 			}
-			// If url isn't the same as last one - send notification
+			// If url isn"t the same as last one - send notification
```

Comment:
nit: typo?

Reply:
https://www.google.com/search?q=isn%27t&tbs=cdr:1,cd_min:1/1/0
Pretty sure abbreviations are with `'` but ok this doesn't matter lol 😆